locals.tf

```diff
@@ -0,0 +1,35 @@
+locals {
+  tags = merge(
+    {
+      "terraform-module" = "network"
+      "terraform"        = "true"
+      "cluster-name"     = var.name
+    },
+    var.tags
+  )
+  intial_node_pool_instance_count = var.control_plane ? 2 : 1
+  node_pools = {
```

Comment:
Can we add provision for additional agent pools in the module? Right now we will have to go around terraform if we need an additional agent pool it seems